collectors/proc.plugin/README.md

```diff
@@ -322,6 +323,46 @@ each state.
 
 By default Netdata will enable monitoring metrics only when they are not zero. If they are constantly zero they are ignored. Metrics that will start having values, after Netdata is started, will be detected and charts will be automatically added to the dashboard (a refresh of the dashboard is needed for them to appear though).
 
+### Monitoring wireless network interfaces
+
+The settings for monitoring wireless is in the `[plugin:proc:/proc/net/wireless]` of your `netdata.conf` file.
+
+```conf
+    status for all interfaces = yes
+    quality for all interfaces = yes
+    discarded packets for all interfaces = yes
+    missed beacon for all interface = yes
+```
+
+For all configuration options:
+
+-   `auto` = enable monitoring if the collected values are not zero
+-   `yes` = enable monitoring
+-   `no` = disable monitoring
+
+#### Monitored wireless interface metrics
+
+-   **Status**
+    The current state of the interface. This is a device-dependent option.
+
+-   **Link**    
+    Overall quality  of  the  link. 
+
+-   **Level**
+    Received signal strength (RSSI), which indicates how strong the received signal is.
+    
+-   **Noise**
+    Background noise level.    
+    
+-   **Discarded packets**
+    
+   Discarded packets for: Number of packets received with a different NWID or ESSID (`nwid`), unable to decrypt (`crypt`), hardware was not able to properly re-assemble the link layer fragments (`frag`), packets failed to deliver (`retry`), and packets ost in relation with specific wireless operations (`misc`). 
```

Comment:
The formatting for this one looks a little different. It might render fine in the end, but let's make sure it's clean.

```suggestion
-   **Discarded packets**
    Discarded packets for: Number of packets received with a different NWID or ESSID (`nwid`), unable to decrypt (`crypt`), hardware was not able to properly re-assemble the link layer fragments (`frag`), packets failed to deliver (`retry`), and packets ost in relation with specific wireless operations (`misc`). 
```

Reply:
> and packets ost

i guess it is `lost`